Composer/packages/client/src/components/BotRuntimeController/BotController.tsx

```diff
@@ -209,60 +210,63 @@ const BotController: React.FC<BotControllerProps> = ({ onHideController, isContr
         return <BotRuntimeStatus key={projectId} projectId={projectId} />;
       })}
       <div ref={botControllerMenuTarget} css={[startPanelsContainer]}>
-        <DefaultButton
-          primary
-          aria-roledescription={formatMessage('Bot Controller')}
-          ariaDescription={startPanelButtonText}
-          data-testid={'startBotButton'}
-          disabled={disableStartBots || areBotsStarting}
-          iconProps={{
-            iconName: statusIconClass,
-            styles: {
-              root: {
-                color: `${NeutralColors.white}`,
-              },
-            },
-          }}
-          id={'startbot'}
-          menuAs={() => null}
-          styles={{
-            root: {
-              backgroundColor: CommunicationColors.tint10,
-              display: 'flex',
-              alignItems: 'center',
-              minWidth: '229px',
-              height: '36px',
-              flexDirection: 'row',
-              padding: '0 7px',
-              border: `1px solid ${CommunicationColors.tint10}`,
-              width: '100%',
-            },
-            rootHovered: {
-              background: transparentBackground,
-            },
-            rootDisabled: {
-              opacity: 0.6,
-              backgroundColor: CommunicationColors.tint10,
-              color: `${NeutralColors.white}`,
-              border: 'none',
-              font: '62px',
-            },
-          }}
-          title={startPanelButtonText}
-          onClick={handleClick}
+        <PVADisableFeature
+          content={formatMessage('PVA bots cannot be run at the moment. Publish the bot to PVA and test it there.')}
```

Comment:
nit: This could be a constant that gets imported so we only have to change the copy in one place if necessary